addon/content/es-modules/components/message-tags.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/*
+ * This is a temporary wrapper. XUL doesn't allow es-modules, but eventually
+ * all components should be exported as es-modules. This code "re-exports"
+ * globals as es-modules, which enables them to be imported (e.g. for tests)
+ * or used as globals.
+ *
+ * When the switch to a WebExtension is done, the actual code should be migrated
+ * here and the global variable workarounds should be removed.
+ */
+
+/* globals require */
```

Comment:
```suggestion
/* eslint-env node */
```

Reply:
When everything is switched to an es module, `require` will no longer be needed and this code will be used in the extension. So I guess we're left with the choice of deleting a global declaration or deleting an environment declaration. My preference would be the global declaration. Do you think the other way around?

Reply:
No, that makes sense, lets leave it as-is.